app/api/custom/tickets.py

```diff
@@ -0,0 +1,22 @@
+from flask import Blueprint, jsonify
+
+from app.api.attendees import get_sold_and_reserved_tickets_count
+from app.models.ticket import Ticket
+
+tickets_routes = Blueprint('tickets_routes', __name__, url_prefix='/v1/events')
```

Comment:
Instead of new file, move this to api/tickets

Reply:
Okay

Reply:
Please check now, I am not sure if this is the required result.